paasta_tools/mesos_tools.py

```diff
@@ -655,21 +655,10 @@ def terminate_framework(framework_id):
     resp.raise_for_status()
 
 
-def get_container_name(app_id, slave_hostname=None, task_id=None):
+def get_tasks_from_app_id(app_id, slave_hostname=None, task_id=None):
```

Comment:
Mostly because I call `sys.exit` from it so I don't want non cli use. I guess it would be better to raise an exception, then I could catch it in the cli clients and call the `sys.exit` there.